.maintain/monitoring/alerting-rules/alerting-rules.yaml

```diff
@@ -73,24 +73,33 @@ groups:
   # Transaction queue
   ##############################################################################
 
-  - alert: TransactionQueueSize
-    expr: 'polkadot_sub_txpool_validations_scheduled -
-    polkadot_sub_txpool_validations_finished > 10'
+  - alert: TransactionQueueSizeIncreasing
+    expr: 'increase(polkadot_sub_txpool_validations_scheduled[5m]) -
```

Comment:
Should `polkadot_sub_txpool_validations_invalid` be taken into account too? Maybe `increase(polkadot_sub_txpool_validations_scheduled[5m]) - increase(polkadot_sub_txpool_validations_finished[5m]) - increase(polkadot_sub_txpool_validations_invalid[5m]) > 0` WDYT?